p2p/libp2p/netMessenger.go

```diff
@@ -55,21 +57,30 @@ var maxSendBuffSize = (1 << 20) - messageHeader
 var log = logger.GetOrCreate("p2p/libp2p")
 
 var _ p2p.Messenger = (*networkMessenger)(nil)
+var externalPackages = []string{"dht", "nat", "basichost", "pubsub"}
+
+func init() {
+	for _, external := range externalPackages {
+		_ = logger.GetOrCreate(fmt.Sprintf("p2p/libp2p/external/%s", external))
```

Comment:
you could use + instead of fmt.SprintF "p2p/libp2p/external/" + external

Reply:
done